tests/system/conftest.py

```diff
@@ -1040,3 +1054,53 @@ def floats_bf(session, floats_pd):
 @pytest.fixture()
 def floats_product_bf(session, floats_product_pd):
     return session.read_pandas(floats_product_pd)
+
+
+@pytest.fixture(scope="session", autouse=True)
+def cleanup_cloud_functions(session, cloudfunctions_client, dataset_id_permanent):
+    """Clean up stale cloud functions."""
+    permanent_endpoints = get_remote_function_endpoints(
+        session.bqclient, dataset_id_permanent
+    )
+    delete_count = 0
+    for cloud_function in get_cloud_functions(
+        cloudfunctions_client,
+        session.bqclient.project,
+        session.bqclient.location,
```

Comment:
Don't we create functions in more than one location?

Reply:
Nope, one cloud function for one BQ location. For multi-region we map US -> us-central1 and EU -> europe-west1 https://github.com/googleapis/python-bigquery-dataframes/blob/adadb0658c35142fed228abbd9baa42f9372f44b/bigframes/functions/remote_function.py#L72